angularSTEP/src/app/navbar/navbar.component.scss

```diff
@@ -1,22 +1,19 @@
-.spacer{
-    flex: 1 1 auto;
-}
-
-a:hover{
-    text-decoration: none;
-    color: red;
-}
-
-a:active{
-    text-decoration: none;
-    color: red;
-}
-
-a:visited{
-    text-decoration: none;
-    color: red;
+.search-input {
+    font: 400 14px/20px Roboto, "Helvetica Neue", sans-serif;
+    font-size: 16px;
+    background: #f5f5f5;
+    color: rgba(0, 0, 0, 0.87);
+    border: 1.5px solid rgba(0, 0, 0, 0.87);
+    outline: none;
+    height: 35px;
+    border-radius: 10px;
+    padding: 5px 12px;
+    margin: 0;
+    width: 200px;
+    vertical-align: bottom;
+    text-align: inherit;
 }
 
-a{
-    border-radius: none;
-}
\ No newline at end of file
+.example-spacer {
+    flex: 1 1 auto;
```

Comment:
I'll admit I'm not familiar with `flex` but it's surprising to see non-zero numeric values without a unit attached. Should these have a `px` or something?

Reply:
Mentioned above, but here is documentation: https://www.w3schools.com/cssref/css3_pr_flex.asp
Also looks weird to me!